Milestone #5

lucasmazo32 opened the pull request:
The live version for this project is not available yet since pushing the code must be done from the master branch.

I hope you enjoy reviewing the project!

PS: I had big trouble understanding the next - total way of thinking from microverse requirements. What I did instead was to work only with the displayed string (I used the state next for this) and used regular expressions to navigate my way through the program. I left the state total in case I want to include a time travel feature in the future.

Thanks!

lucasmazo32 committed:
Finished the calculator logic and is working as expected

lucasmazo32 committed:
Finished the README file and fixed division by 0 error

shloch reviewed (changes requested):
## Hello @lucasmazo32 , 

WOrks perfectly well

Before you get final thumbs up, ONE more task needed please. It's about bringing a number back into calculation after changing it's sign.

For example: if i want to calculate `-8 + 6`, first, i have to press `8`, then i press `+/-` to negate it. Once i do that, i can't press `+` again.

Make sure that when a number's sign is pressed, it can be used in a calculation

### Happy coding

lucasmazo32 committed:
Worked with negatives and the possibility of starting with a negative sign

lucasmazo32 committed:
solved possibility of having consecutive signs

lucasmazo32 commented:
- Added functionality to start with a negative sign and then you can add other operators.
- Fixed slicing method so the program works as expected when including multiple signs in the same operation.

lucasmazo32 committed:
solved multiple negative signs at the end of the ecuation

lucasmazo32 committed:
solved lintern errors

shloch reviewed (approved):
## hello @lucasmazo32 , 
Good to go
![Screen Shot 2020-04-26 at 8 17 11 PM](https://user-images.githubusercontent.com/40712459/80317235-7acc3b00-87fa-11ea-8588-e9322d1bd52b.png)

You can merge this
